code/modules/surgery/organs/lungs.dm

```diff
@@ -388,6 +388,14 @@
 			if(prob(20))
 				to_chat(H, "<span class='warning'>You feel [hot_message] in your [name]!</span>")
 
+	// the air you breath out should match your body tempurature
+	if(H.bodytemperature > breath.temperature)
+		// raise the tempurature of our breath to just below body temp
```

Comment:
```suggestion
		// raise the temperature of our breath to just below body temp
```

Reply:
I have fixed all the changes, and reworded a bit.